scripts/yum_clean_all

```diff
@@ -0,0 +1,12 @@
+#! /bin/sh -eu
+
+# (centos:6 only) If glibc-common is updated, we have to manually purge non-en_US locales.
+if grep -F 'CentOS release 6' /etc/centos-release ; then
+    if locale -a | grep nds_DE ; then
+        cp /usr/lib/locale/locale-archive /usr/lib/locale/locale-archive.tmpl
```

Comment:
(`cp` instead of `mv` to avoid writing to the mmapped file later.)

Reply:
Should we make that a comment in the script?

Reply:
Nah, not really important. The only change I expect to happen to this code is to be removed whenever this isn't needed anymore, e.g., when we use Centos 7 everywhere.
I just added the comment for my future self if I ever wondered why I changed that.